src/pages/experiments/[experimentId]/plots-and-tables/frequency/index.jsx

```diff
@@ -44,7 +45,10 @@ const FrequencyPlotPage = ({ experimentId }) => {
     loading: cellSetsLoading,
     error: cellSetsError,
   } = cellSets;
-
+  const louvainClusters = useSelector(getCellSetsHierarchyByKeys('louvain'));
+  const experimentName = useSelector((state) => state.experimentSettings.info.experimentName);
+  const [csvData, setCsvData] = useState([]);
+  const [csvFilename, setCsvFilename] = useState('');
   useEffect(() => {
```

Comment:
Can you put some spacing between the lines of code? Putting spacing between similar group of codes make it easier to read,